todolist/src/components/TodoContainer.js

```diff
@@ -3,84 +3,164 @@ import TodosList from "./TodosList";
 import Header from "./Header"
 import InputTodo from "./InputTodo"
 import { v4 as uuidv4 } from "uuid";
+// class TodoContainer extends React.Component {
+//     state = {
```

Comment:
In this part of the code, it would better to use ES6 syntax for this. Here is what I mean;
```const TodoContainer = () => {}```
This  will improves code readability, better code organisation and even better performance.